proxy/Main.cc

```diff
@@ -185,6 +185,8 @@ static ArgumentDescription argument_descriptions[] = {
   {"httpport", 'p', "Port descriptor for HTTP Accept", "S*", &http_accept_port_descriptor, "PROXY_HTTP_ACCEPT_PORT", nullptr},
   {"dprintf_level", 'o', "Debug output level", "I", &cmd_line_dprintf_level, "PROXY_DPRINTF_LEVEL", nullptr},
   {"disable_freelist", 'f', "Disable the freelist memory allocator", "T", &cmd_disable_freelist, "PROXY_DPRINTF_LEVEL", nullptr},
+  {"disable_fl_more", 'F', "Disable the freelist memory allocator in ProxyAllocator", "T", &cmd_disable_freelist_more,
+   "PROXY_DPRINTF_LEVEL", nullptr},
```

Comment:
The more I see this, I find this naming and description very unfortunate. At least call it for what it is, not "more".

So, e.g. --disable_thread_freelist.